### ISSUE TITLE
How to set MY font to UI?

### ISSUE THREAD

**Issue by exec228** [a85/POSTMan-Chrome-Extension#12240]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

Hello!

The problem is simple and clear: this application has no font configuration.
Why this is problem? Because Chrome, therefore, Electron just cannot into fonts. Never could, never will be able.

![](https://i.imgur.com/xqMRRcu.png)

### Describe the solution you'd like

Here I ask, beg, request or demand to introduce a UI font setting for Postman.
Please, please, please

### Describe alternatives you've considered

I extracted everything from «resources\app.asar».
I replaced all mentions «font-family:» in all .css, .html, .js with «font-family: Verdana».
I packed everything back to «resources\app.asar».
Still, the font not changed.

### Additional context

Please, do not ask me to tune my display and OS. It's already in the state I like for most applications. Or else my eyes will be bleeding.

**Comment by exec228** [comment]
I have activated «View\Developer\Show Dev Tools» and found out that the hated Inter font is addressed from `https://desktop.postman.com/_ar-assets/styles/base/_typography.scss` which is not located on my PC.
This file is forbidden to view directly from a browser.

Now, this issue is even worse, since editing app.asar is not effective any more, indeed.

Please-please-please-please-please, make a desktop application **setting to choose a custom UI font**!

**Comment by amrindersandhu** [comment]
@exec228 Thanks for bringing this up. Can you please try removing the `font-smoothing` CSS styles and see it that fixes the issue?

<img width="435" alt="image" src="https://github.com/postmanlabs/postman-app-support/assets/910811/03872466-bf2d-457e-8e74-e7e501e86aac">

Or, you can also pick a time that works for you so I can see the issue more closely and help fix it. https://calendly.com/d/4cc-jtz-5kk/fix-fonts

**Comment by amrindersandhu** [comment]
@exec228 Checking back on this. Were you able to resolve the issue? Did the above solution work?

**Comment by exec228** [comment]
@amrindersandhu Hi! I wrote a direct email to you

**Comment by amrindersandhu** [comment]
@exec228 Thanks! Unfortunately, the email on my GitHub profile was not updated, and I no longer have access to that email.  Can you please email me at amrinder@outlook.in?

**Comment by exec228** [comment]
Have you received an email on 2023-10-04?

**Comment by amrindersandhu** [comment]
@exec228 Yes, I will revert to it today and, if required, set up a call between us early next week.

**Comment by exec228** [comment]
> Informations de diagnostic pour les administrateurs :
> Serveur de génération : SA1P*********.namprd10.prod.outlook.com
> *********@hey.com
> home-mx.app.hey.com
> Remote server returned '550 5.1.1 <*********@hey.com>: Recipient address rejected: User unknown'

I have this response twice

**Comment by StudentCWZ** [comment]
I also have needs. I want to change the software font to Monaco instead of just changing the editor. I hope that related options for changing UI font will be provided in the future.

**Comment by amrindersandhu** [comment]
@StudentCWZ Can you please share why you need to change the app UI font? This will help us with understanding the problem and come up with a better solution.

**Comment by StudentCWZ** [comment]
Because everyone will have their own favorite and commonly used programming fonts. It might be better if there is an option to change the UI font so that each user can choose the font they like to use.
Postman is a software I often use. For me, I still hope that postman can provide an option to change the UI font. @amrindersandhu

**Comment by exec228** [comment]
I will add my penny:

1) MSWindows has non-optimal anti-aliasing. All fonts seem unfocused and blurred. My eyes are bleeding.

2) I switched off anti-aliasing on my MSWindows totally. Most fonts look very scary (screenshot attached above). My eyes are bleeding.

3) My solution is... Pixel perfect fonts! What are these? By Joel Spolsky's excellent description, it is a simple MS solution for raster displays. Just make fonts **fit pixels**. And MS indeed designed and released "MS Web Core fonts". As simple as it is! This collection has the **reading-pleasant** (NOTA BENE!) `Verdana`, and the optimal-looking `Courier New`, to name a few.

4) Actually, there are very few pixel perfect TrueType fonts in the wild. `PT Mono` is a **very** rare example of one.

5) Eventually, I configured almost all applications with `Verdana/PT Mono`. Including MSWindows itself.

6) And I encounter plenty of Electron shlocks, with `InterSegoeIBMPlexMono` nightmare fonts hardcoded and **welded** into it.
I can only tear app.asar apart, chase and scorch out these fonts from all CSS embedded, to pack app.asar back with fonts specified **by me**.

And now, out of the blue, Postman tries to evade this procedure, by **hot-loading online CSS** with these hateful fonts!

As Electron is the ugly Chrome (Chrome works with fonts even worse than MSWindows does), it supports `--disable-remote-fonts` command line argument. It should force Postman to use system (my system, `Verdana` and `PT Mono`) fonts, **but** all button glyphs are destroyed as well.

One must invent some way to substitute online resources by offline ones, which can be **fixed** by one's taste.

Or we just can convince Postman Developers to add an option for custom GUI fonts.

**Comment by exec228** [comment]
> @exec228 Checking back on this. Were you able to resolve the issue? Did the above solution work?

So, the designers team is happy with these illegible fonts?
![TzS3FSr 1](https://github.com/postmanlabs/postman-app-support/assets/275328/dc09baa8-d1d1-40df-aff0-4e65293e4c5e)

**Comment by capegreg** [comment]
UWP apps include Ctrl or CMD zoom+/- to change the IDE font size. Can this be added?
